CONTRIBUTING.md

```diff
@@ -207,9 +211,74 @@ To build the documentation locally:
 
 1. Navigate to the `docs` directory.
 2. Run `make html`.
-3. Open "_build/html/index.html" in your browser.
+3. Open "\_build/html/index.html" in your browser to inspect the documentation.
 
 Note that `nbsphinx` currently requires the stand-alone program `pandoc`. If you
-get an error "Pandoc wasn't found", install that pandoc as described in
+get an error "Pandoc wasn't found", install `pandoc` as described in
 [nbsphinx installation
 guide](https://nbsphinx.readthedocs.io/en/0.9.2/installation.html#pandoc).
+
+### Jupyter notebook style guide
```

Comment:
Nitpick: I would suggest bumping the heading level of this section up one level, i.e.:

```
## Jupyter notebook style guide
### General jupyter guidelines
et cetera
```

The reason being, we might wish to put these contributing guidelines on Sphinx at some point in future, and with the ReadTheDocs theme it's generally helpful for the layout if there are only two main levels of headings used throughout each document, rather than 3 or more levels of nesting.